test/unit/plugins/kernel_v2/config/vm_test.rb

```diff
@@ -214,12 +223,22 @@ def assert_valid
         guest: 22, host: 14100, id: "winrm"
       subject.finalize!
 
-      n = subject.networks
-      expect(n.length).to eq(2)
-      expect(n[0][0]).to eq(:forwarded_port)
-      expect(n[0][1][:guest]).to eq(22)
-      expect(n[0][1][:host]).to eq(14100)
-      expect(n[0][1][:id]).to eq("winrm")
+      winrm_network = find_network 'winrm'
+      expect(winrm_network[:guest]).to eq(22)
+      expect(winrm_network[:host]).to eq(14100)
+      expect(winrm_network[:id]).to eq("winrm")
+    end
+
+    it "allows overriding WinRM SSL" do
+      subject.communicator = :winrmssl
```

Comment:
You're right... it should just be :winrm.

The test is supposed to be for the second network forwarded for winrm, though. See https://github.com/maxlinc/vagrant/blob/winrmssl/plugins/kernel_v2/config/vm.rb#L403-L408.

Also your the discussion on https://github.com/mitchellh/vagrant/issues/4932, including your response:
> It seems logical to only automatically forward networks as required by the active communicator. I don't think its currently breaking anything by automatically forwarding both, but its sloppy.

I don't care if it's zero (the cloud providers don't use forwarded ports, AFAIK), one (cleaner) or two (easier to switch - because I changing forwarded ports requires a reload). I'll fix the communicator to be winrm though.